imap_processing/lo/l0/lo_science.py

```diff
@@ -70,7 +71,11 @@ def parse_histogram(dataset: xr.Dataset, attr_mgr: ImapCdfAttributes) -> xr.Data
     dataset : xr.Dataset
         Parsed and decompressed histogram data.
     """
-    hist_bin = dataset.sci_cnt
+    # TODO: improve this as needed
+    binary_str_val = []
+    for data in dataset.sci_cnt.values:
+        binary_str_val.append(convert_to_binary(data))
+    hist_bin = binary_str_val
```

Comment:
list comprehensions are faster and easier to read too without needing multiple variables.
```suggestion
    hist_bin = [convert_to_binary(data) for data in dataset.sci_cnt.values
```